### PR TITLE
Added Expandable Row for table component

### PR DESCRIPTION
**Pull request by vinodrxnt**
Created Expandable row for table component. We have added **isTableExpandable** in table object in uischema to display Expandable row

### CODE REVIEW

File: src/table/index.js [hunk 1]
```diff
@@ -68,17 +68,23 @@ class TableField extends Component {
     this.handleDeletedRow = this.handleDeletedRow.bind(this);
     this.handleRowSelect = this.handleRowSelect.bind(this);
     this.handleAllRowSelect = this.handleAllRowSelect.bind(this);
+    this.checkMyRow = this.checkMyRow.bind(this);
```
Comment (sneu012) [inline src/table/index.js:71]: checkMyRow? Does my stand for anything here? A better name would definitely help.

Comment (sneu012) [reply src/table/index.js:71]: Similar issue in the next line too.

File: src/table/index.js [hunk 2]
```diff
@@ -212,19 +228,56 @@ class TableField extends Component {
       schema,
       uiSchema,
       registry,
-      version: "1",
+      version: "1"
     };
     return <InsertModal {...attr} />;
   };
+  checkMyRow(isTableExpandable) {
```
Comment (sneu012) [inline src/table/index.js:235]: Is checkMyRow checking to see if the rowIsExpandable?

Comment (vinodrxnt) [reply src/table/index.js:235]: Yes

File: src/table/index.js (hunk 1 above)
Comment (vinodrxnt) [reply src/table/index.js]: updated function name to **isRowIsExpandable**

Comment (sneu012) [reply src/table/index.js]: How about isRowExpandable?

**Review by sneu012 — APPROVED** [review]

**Pull request merged**